### PR TITLE
Ensure installed docker version is not uninstalled by keadm if the version is equal to or greater than 17.03.1 docker version

### PR DESCRIPTION
**Pull request by naveensriram**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
 /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:
Though latest version of docker was installed in the system such as 18.09.2 keadm earlier used to uninstall that version and install 18.06.0 version. 
This fix validates against the installed docker version is greater than or equal to 17.03.1 and only in case of lower than 17.03.1 version keadm installs docker version.

 
**Which issue(s) this PR fixes**:
No specific issue is created prior on this.

**Special notes for your reviewer**:

Among the qualified docker versions mentioned (please refer to below mentioned reference link), docker version 1.13.1 is not shown in the output 'apt-cache madison 'docker-ce'' next mentioned & available docker version is 17.03.1 treated as the minimum docker version 
Reference - https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#external-dependencies

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by naveensriram** [comment]
CC @sids-b @subpathdev

**Comment by naveensriram** [comment]
cc @fisherxu

File: keadm/app/cmd/util/ubuntuinstaller.go [hunk 1]
```diff
@@ -146,6 +147,51 @@ func (u *UbuntuOS) IsDockerInstalled(defVersion string) (types.InstallState, err
...
 
+	if str != "" {
+		fmt.Println("Docker Version installed is = ", str)
+	}
+
+	// Docker version format MM.mm.PP
+	// Split version number into three parts M = Major number, m = minor number, P = patch number
+	sliceOfDockerMinimumVersionNumber := strings.SplitN(DockerMinimumVersionNumber, ".", 3)
+
+	minimumDockerMajorVersionNumber, _ := strconv.Atoi(sliceOfDockerMinimumVersionNumber[0])
```
Comment (subpathdev) [inline keadm/app/cmd/util/ubuntuinstaller.go]: please check the errors

**Comment by subpathdev** [comment]
please check ut

**Comment by fisherxu** [comment]
will close this first :)
/close

**Pull request closed without merging**